app/src/main/java/eu/kanade/tachiyomi/data/track/Tracker.kt

```diff
@@ -82,4 +84,8 @@ interface Tracker {
     suspend fun setRemoteStartDate(track: Track, epochMillis: Long)
 
     suspend fun setRemoteFinishDate(track: Track, epochMillis: Long)
+
+    suspend fun setRemotePrivate(track: Track, private: Boolean)
+
+    suspend fun togglePrivate(track: Track) = this.setRemotePrivate(track, !track.private)
```

Comment:
toggle only happens in one single place so no need to make it a function of the tracker